### ISSUE TITLE
Anyway to update the delta as the loop goes on?

### ISSUE THREAD

**Issue by FantomHD** [tangmi/node-gameloop#2]
If I have a CPU intensive task being run in the loop, the delta time doesn update fromt he start of the loop till the end. Anyway it can be updated throughout the loop or be able to run a check to get a second new delta of the time that passed since the loops delta was called, if that makes sense? To add on to the loop's delta any time that was taken inside of one tick of the loop?

**Comment by tangmi** [comment]
I'm not entirely sure what the question is, but if you'd like to change the target delta time, one can simply make a new game loop with the new target delta time (and clear the previous game loop).

If you are running into issues where a CPU intensive task is making your loop exceed the target delta time, you can consider using [child_process.fork()](https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options) to do that work in another process and asynchronously consume it in another "frame".

**Comment by FantomHD** [comment]
I just realized I never replied to this comment. Sorry for the super late reply, but I saw your comment then and fixed my issue. What I meant was, inside the loop I had an intensive task that took a bit to finish, so the delta time wasn't useful for what I needed it for. Like the delta time was constant throughout the top of the loop to the bottom, and what was happening was I had a intensive task in the loop that made the 'real' time from that point I was measuring delta time from the end of the previous loop, to actually be longer than delta time was. It's hard to explain but if I had the old code I could show you and you would instantly understand, but I dont and forking it fixed my issues.
